documentation/doc.css

```diff
@@ -43,19 +40,74 @@ iframe {
   flex-wrap: wrap;
 }
 
-.input-box-section {
-  display: flex;
-  justify-content: flex-end;
-  width: 50%;
+/**/
+
+.doc-page {
+  /* background-color: aqua; */
+  display: grid;
+  gap: 1rem;
+  grid-template-areas:
+    "header header header header"
+    "aside main main main"
+    "aside main main main"
+    "aside main main main";
```

Comment:
you can do this instead:

header header
aside main

Reply:
Done the changes